sway/desktop/render.c

```diff
@@ -39,6 +39,8 @@ struct render_data {
 struct decoration_data get_undecorated_decoration_data() {
 	return (struct decoration_data) {
 		.alpha = 1.0f,
+		.dim = 0.0f,
```

Comment:
Wouldn't we want this to be 1.0f?

Reply:
It's a bit confusing... We want the value to default to 0.0, otherwise the `mix` method in the GLSL would apply the diming. That's why I subtract the config value from 1.0 in the other locations

Reply:
Ah I see. I'm wondering if we should change this from `dim` to `inactive_brightness`, set it to 1.0 here and skip the subtraction elsewhere, and then in fx_renderer.c convert from the brightness value to whatever we need to dim appropriately, be it subtraction / whatever else. Right now it's a little confusing given the wording. Open to your thoughts here too though!